util/globalconn/globalconn.go

```diff
@@ -46,4 +62,217 @@ const (
 	LocalConnIDBits64 = 40
 	// MaxLocalConnID64 is maximum localConnID for 64bits global connection ID.
 	MaxLocalConnID64 = 1<<LocalConnIDBits64 - 1
+
+	// ReservedCount is the count of reserved connection IDs for internal processes.
+	ReservedCount = 200
 )
+
+// ToConnID returns the 64bits connection ID
+func (g *GCID) ToConnID() uint64 {
+	var id uint64
+	if g.Is64bits {
+		id |= 0x1
```

Comment:
How about throwing errors when g.LocalConnID exceeds MaxLocalConnID64 (instead of silently ignores the overflowed part)?

Reply:
It's changed to check and panic on overflow. As the overflow should not happen unless there is a bug.

PTAL~